### PR TITLE
NLPCC14-SC Support！

### PR DESCRIPTION
**Pull request by fiyen**

### CODE REVIEW

**Review by ZeyuChen — APPROVED** [review]
LGTM! Thanks for your contribution!

**Comment by ZeyuChen** [comment]
@smallv0221 Please review the data contribution, thanks. Merge this PR after your approval.

**Comment by ZeyuChen** [comment]
Resolve #495 for dataset contribution.

**Comment by ZeyuChen** [comment]
@fiyen Please edit the PR and add more message if possible.

**Review by smallv0221 — COMMENTED** [review]
Thanks for your contribution. There are only two samll advices:
1. It is better to provide more information of this dataset for others to use it. At least declear the task of this dataset.
2. Always use camel case for the class name of DatasetBuilder. In this case the classname should be NLPCC14SC.

File: paddlenlp/datasets/nlpcc14_sc.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+from paddle.utils.download import get_path_from_url
+from paddlenlp.utils.env import DATA_HOME
+from . import DatasetBuilder
+
+__all__ = ['NLPCC14_SC']
+
+
+class NLPCC14_SC(DatasetBuilder):
+    """
+    NLPCC14-SC datasets only contains train and test data, so we remove the dev
```
Comment (smallv0221) [inline paddlenlp/datasets/nlpcc14_sc.py:28]: You can declear task here.

File: paddlenlp/datasets/nlpcc14_sc.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+from paddle.dataset.common import md5file
+from paddle.utils.download import get_path_from_url
+from paddlenlp.utils.env import DATA_HOME
+from . import DatasetBuilder
+
+__all__ = ['NLPCC14_SC']
+
+
+class NLPCC14_SC(DatasetBuilder):
```
Comment (smallv0221) [inline paddlenlp/datasets/nlpcc14_sc.py:26]: Change class name to camel case

**Comment by ZeyuChen** [comment]
We will merge the code first then refine the naming style in next PR. @fiyen Thanks for your contribution!

**Pull request merged**